spec/views/posts/index.html.erb_spec.rb

```diff
@@ -1,5 +1,22 @@
 require 'rails_helper'
 
-RSpec.describe 'posts/index.html.erb', type: :view do
-  #  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.feature 'User Post Index Page' do
+  scenario 'Viewing user posts' do
+    # Create a test user directly in the test database
+    user = User.create(name: 'Test User', bio: 'Test Bio')
+    
+    # Create posts associated with the user
+    post1 = user.posts.create(title: 'Post 1', text: 'Text 1')
+    post2 = user.posts.create(title: 'Post 2', text: 'Text 2')
+    post3 = user.posts.create(title: 'Post 3', text: 'Text 3')
+
+    visit user_posts_path(user)
```

Comment:
- [ ] It would be better if we fix the failing test cases as seen below
![Screenshot from 2023-09-20 16-16-06](https://github.com/Gardimy/BlogApp/assets/46388113/3481a947-7e2f-49b9-a0ce-391f68eafecd)